backend/app/context_processors.py

```diff
@@ -28,5 +28,15 @@ def additional_settings_export(request):
     # per-request syndicate overrides the global setting. This is so that Mintion users can use Obico cloud but see their own theme.
     syndicate_header = request.META.get('HTTP_X_OBICO_SYNDICATE', None)
     if syndicate_header:
-        settings_dict['syndicate'] = {"provider": syndicate_header}
+        settings_dict["syndicate"] = {"provider": syndicate_header,"brand_name": brand_name}
+
+    accept_language_header = request.META.get("HTTP_ACCEPT_LANGUAGE", "")
```

Comment:
Like I commented in TSDApp PR, please check if we can just use Accept-Language to pass the language selection to backennd